doc/readme.md

```diff
@@ -40,3 +40,54 @@ RAILS_ENV=production bundle exec rake assets:precompile
 ```
 bundle exec thin -p 8080 -e production -d
 ```
+
+How to setup development envirement
+------------------
+
+* install ruby 1.9.3
```

Comment:
Just made a new commit in the branch, as my understanding, I shouldn't need a new pulling request.